bin/hdfs-mesos-namenode

```diff
@@ -58,4 +39,4 @@ while getopts ":ibs" opt; do
   esac
 done
 
```

Comment:
If we remove the sleep loop, we need to make sure that we are attempting to restart our process repeatedly in our Executors. There are many things that can go wrong, which is why we keep trying to restart the process here, i.e. sometimes upon NN1 failure, we need to keep trying to restart until we're able to connect NN2 with NN1.